README.md

```diff
@@ -108,6 +108,10 @@ cluster after reboot. Consult your distribution's documentation on this. If you
 uses packages provided by your distribution this should be automatic (or a 
 simple configuration change)
 
+How to make scheduler persistent:
+
+By adding an option --scheduler-host for daemon and --persistent-client-connection for scheduler
```

Comment:
If seems like this belongs elsewhere in this file, or just in a section not related to "make scheduler persistent"